<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-2B05C2G" failures="1" tests="6" name="Amazon1.TC_3AmazonLoginFail" time="134.321" errors="1" timestamp="2025-06-21T14:57:47 EDT" skipped="4">
  <testcase classname="Amazon1.TC_3AmazonLoginFail" name="loginWithInvalidEmail" time="16.839">
    <skipped/>
    <system-out>
      <![CDATA[Invalid Username is entered]]>
    </system-out>
  </testcase> <!-- loginWithInvalidEmail -->
  <testcase classname="Amazon1.TC_3AmazonLoginFail" name="loginWithInvalidEmail" time="16.343">
    <skipped/>
    <system-out>
      <![CDATA[Invalid Username is entered]]>
    </system-out>
  </testcase> <!-- loginWithInvalidEmail -->
  <testcase classname="Amazon1.TC_3AmazonLoginFail" name="loginWithInvalidEmail" time="23.118">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at Amazon1.AmazonHomeLoginPage.verifyAccountInvalid(AmazonHomeLoginPage.java:191)
at Amazon1.TC_3AmazonLoginFail.loginWithInvalidEmail(TC_3AmazonLoginFail.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
    </failure>
    <system-out>
      <![CDATA[Invalid Username is entered
Invalid Password is entered
Exception is Handeled
Test case failed: loginWithInvalidEmail]]>
    </system-out>
  </testcase> <!-- loginWithInvalidEmail -->
  <testcase classname="Amazon1.TC_3AmazonLoginFail" name="loginWithInvalidEmail" time="15.169">
    <error message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//input[@id=&#039;ap_password&#039;]&quot;}
  (Session info: chrome=137.0.7151.119)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &#039;4.33.0&#039;, revision: &#039;2c6aaad03a&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.12&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [4a0365730b4832bc8552eff64c49ef7e, findElement {using=xpath, value=//input[@id=&#039;ap_password&#039;]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.119, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: C:\Users\ganesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:65298}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:65298/devtoo..., se:cdpVersion: 137.0.7151.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 4a0365730b4832bc8552eff64c49ef7e" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@id='ap_password']"}
  (Session info: chrome=137.0.7151.119)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.33.0', revision: '2c6aaad03a'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.12'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [4a0365730b4832bc8552eff64c49ef7e, findElement {using=xpath, value=//input[@id='ap_password']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.119, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: C:\Users\ganesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:65298}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:65298/devtoo..., se:cdpVersion: 137.0.7151.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 4a0365730b4832bc8552eff64c49ef7e
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:544)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:367)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:361)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy13.sendKeys(Unknown Source)
at Amazon1.AmazonHomeLoginPage.pwdinvalid(AmazonHomeLoginPage.java:157)
at Amazon1.TC_3AmazonLoginFail.loginWithInvalidEmail(TC_3AmazonLoginFail.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
    </error>
    <system-out>
      <![CDATA[Invalid Username is entered
Test case failed: loginWithInvalidEmail]]>
    </system-out>
  </testcase> <!-- loginWithInvalidEmail -->
  <testcase classname="Amazon1.TC_3AmazonLoginFail" name="loginWithInvalidEmail" time="31.534">
    <skipped/>
    <system-out>
      <![CDATA[Exception is handaled
Invalid Username is entered
Invalid Password is entered
Exception is Handeled]]>
    </system-out>
  </testcase> <!-- loginWithInvalidEmail -->
  <testcase classname="Amazon1.TC_3AmazonLoginFail" name="loginWithInvalidEmail" time="31.318">
    <skipped/>
    <system-out>
      <![CDATA[Exception is handaled
Invalid Username is entered
Invalid Password is entered
Exception is Handeled]]>
    </system-out>
  </testcase> <!-- loginWithInvalidEmail -->
  <system-out>
    <![CDATA[Test execution started - Listener
Test case failed: registration
Test case failed: registration
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Invalid Username is entered
Chrome browser is launched succesfully
Invalid Username is entered
Chrome browser is launched succesfully
Invalid Username is entered
Test case failed: loginWithInvalidEmail
Chrome browser is launched succesfully
Exception is handaled
Invalid Username is entered
Invalid Password is entered
Exception is Handeled
Chrome browser is launched succesfully
Exception is handaled
Invalid Username is entered
Invalid Password is entered
Exception is Handeled
Chrome browser is launched succesfully
Invalid Username is entered
Invalid Password is entered
Exception is Handeled
Test case failed: loginWithInvalidEmail
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is handaled
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is handaled
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is handaled
Test case failed: loginwithDDT
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Chrome browser is launched succesfully
Exception is Handeled
Test case failed: loginwithDDT
Chrome browser is launched succesfully
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is handeled
Test case failed: loginwithDDT
Chrome browser is launched succesfully
Exception is Handeled
Exception is handeled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handaled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handaled
Product discription is Displayed
Customer Reviews is Displayed
Product price is Displayed
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Test case failed: loginwithDDT
Chrome browser is launched succesfully
Exception is Handeled
Exception is Handaled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handaled
Exception is handaled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is Handeled
Exception is Handaled
Test case passed: loginwithDDT
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handaled
Exception is handaled
Exception is handeled
Test case failed: addRemoveproduct
Chrome browser is launched succesfully
Exception is Handeled
Exception is Handaled
Exception is handeled
Test case passed: addRemoveproduct
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handaled
Exception is handaled
Exception is handeled
Test case failed: selectingAddressPaymentMethod
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handaled
Exception is handeled
Exception is handeled
Test case passed: selectingAddressPaymentMethod
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handaled
Exception is handeled
Test case passed: paymentMethodValidation
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handaled
Exception is handeled
Test case passed: paymentMethodValidation
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Exception is Handaled
Exception is handaled
Exception is handeled
Test case failed: OrderingProuctWithCoupenCode
Chrome browser is launched succesfully
Exception is Handeled
Exception is Handaled
Exception is handeled
Test case failed: OrderingProuctWithCoupenCode
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Exception is Handeled
Exception is Handeled
Test case failed: givingRatingforLastOrder
Chrome browser is launched succesfully
Exception is handaled
Exception is Handeled
Test case passed: givingRatingforLastOrder
Chrome browser is launched succesfully
Exception is handaled
Exception is Handaled
Test case passed: GoingToCartwithoutLogin]]>
  </system-out>
</testsuite> <!-- Amazon1.TC_3AmazonLoginFail -->
